.env.sample

```
@@ -1,5 +1,18 @@
+# Probe Config
+PROBE_TYPE=<SNOWFLAKE | BIGQUERY>
+
 # BigQuery Config
 
 BIGQUERY_PROJECT=<project_name>
 BIGQUERY_DATASET=<dataset_name>
 BIGQUERY_PROJECT_LOCATION=<project_location>
+
+
+# Snowflake Config
+
+SNOWFLAKE_ACCOUNT=<account_name>
+SNOWFLAKE_USERNAME=<user_name>
+SNOWFLAKE_PASSWORD=<password>
+SNOWFLAE_WAREHOUSE=<warehouse_name>
+SNOWFLAKE_DATABASE=<database_name>
+SNOWFLAKE_SCHEMA=<schema_name>
```

```suggestion
SNOWFLAKE_ACCOUNT=
SNOWFLAKE_USERNAME=
SNOWFLAKE_PASSWORD=
SNOWFLAE_WAREHOUSE=
SNOWFLAKE_DATABASE=
SNOWFLAKE_SCHEMA=
```
lets remove placeholder from .env file for all the keys.